src/Monitors/IMUDownlink.cpp

```diff
@@ -0,0 +1,73 @@
+#include "IMUDownlink.hpp"
+
+IMUDownlink::IMUDownlink(unsigned int offset)
+    : TimedControlTask<void>(offset)
+{
+    imu = Adafruit_LSM9DS1(constants::imu::CSAG, constants::imu::CSM);
+    imu.begin();
+    //if(!imu.begin()) then...
+    imu.readGyro();
+    imu.readMag();
```

Comment:
You do not actually need to read any of the imu values in this- it should all be taken care of in IMU Monitor